Symfony/CS/Fixer/Symfony/PhpdocNoPackageFixer.php

```diff
@@ -39,4 +39,15 @@ public function getDescription()
     {
         return '@package and @subpackage annotations should be omitted from phpdocs.';
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        /*
+         * Needs higher priority than any of the fixers cleaning up empty docblocks or fixing headers.
```

Comment:
for example, why are you not allowing the whitespace phpdoc stripping fixer to run if you're allowing the package removal fixer to run